package-lock.json

```diff
@@ -0,0 +1,2401 @@
+{
```

Comment:
package-lock should be added to .gitignore - it stores local preferences for the latest packages. anyone pulling down your code should be relying on package.json